editor/editor_settings.cpp

```diff
@@ -856,7 +856,7 @@ void EditorSettings::save() {
 	Error err = ResourceSaver::save(singleton->config_file_path, singleton);
 
 	if (err != OK) {
-		ERR_PRINT("Can't Save!");
+		ERR_PRINTS("Can't Save! " + singleton->config_file_path);
```

Comment:
Could you rephrase to `"Error saving editor settings to " + <str>`?

Reply:
Yeah, sure, I'll do that in a min.